src/utils/Contract.js

```diff
@@ -66,3 +77,13 @@ export const getBalance = async (walletProvider, walletAddress) => {
     console.log("AckToEarn getBalance transaction failed", err);
   }
 };
+
+export const getWithdrawFunds = async (walletProvider, balance) => {
```

Comment:
nice job hooking up the contract! can we rename this function to `withdrawFunds`?